runtime/debian.rules

```diff
@@ -20,7 +20,4 @@ override_dh_auto_build:
 
 override_dh_auto_install:
 	mkdir -p debian/cc-runtime
-	make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr
-
-
-
+	cd $(GOPATH)/src/github.com/clearcontainers/runtime/ && make install DESTDIR=`pwd`/debian/cc-runtime PREFIX=/usr SYSCONFDIR=/etc
```

Comment:
well, when the make is performed, it changes to another directory, but in order to perform the go build, it needs to be in the gopath.